src/main/java/net/education/kyivstar/config/HikariConnection.java

```diff
@@ -0,0 +1,35 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static net.education.kyivstar.config.HikariConfig.getInstance;
+
+public class HikariConnection {
+    private ConfigDataBase configDataBase;
+    private final static Logger logger = LoggerFactory.getLogger(HikariConnection.class);
+
+    public HikariConnection(ConfigDataBase configDataBase) {
+        this.configDataBase = configDataBase;
+    }
+
+    public Connection connect() {
+        try {
+
+            return getInstance(configDataBase).dataSource().getConnection();
```

Comment:
The getInstance(configDataBase) method is called multiple times in both connect() and closeConnection(). It might be better to call it once and store the result in a variable to improve readability and potentially avoid repeated method calls.